### PR TITLE
Update ci.yml

### PR DESCRIPTION
**Pull request by fschlimb**
@tkarna

### CODE REVIEW

File: conda-recipe/build.sh [hunk 1]
```diff
@@ -27,17 +27,28 @@ popd
...
+    # done
+    # if [ -d "${SPIRV_INC_DIR}" ]; then
+    #     echo "Using SPIRV_INC_DIR=${SPIRV_INC_DIR}"
+    #     mkdir -p "${SRC_DIR}/grrrr/include"
+    #     ln -s "${SPIRV_INC_DIR}" "${SRC_DIR}/grrrr/include/CL"
+    #     SPIRV_INC_DIR="${SRC_DIR}/grrrr/include"
+    # else
+    #     echo "Fatal error: SPIRV_INC_DIR not found"
+    #     exit 1
+    # fi
```
Comment (tkarna) [inline conda-recipe/build.sh:51]: rm comments?

File: conda-recipe/build.sh [hunk 2]
```diff
@@ -83,10 +94,11 @@ if [ ! -d "${INSTALLED_DIR}/imex/lib" ]; then
...
         -DLLVM_EXTERNAL_PROJECTS="Imex"  \
-        -DLLVM_EXTERNAL_IMEX_SOURCE_DIR=. \
-        -DLEVEL_ZERO_DIR=${INSTALLED_DIR}/level-zero \
-        -DIMEX_ENABLE_SYCL_RUNTIME=1 \
-        -DIMEX_ENABLE_L0_RUNTIME=1
+        -DLLVM_EXTERNAL_IMEX_SOURCE_DIR=.
+        # -DLEVEL_ZERO_DIR=${INSTALLED_DIR}/level-zero \
+        # -DIMEX_ENABLE_SYCL_RUNTIME=1 \
+        # -DIMEX_ENABLE_L0_RUNTIME=1 \
+        # -DCMAKE_CXX_FLAGS="-I${SPIRV_INC_DIR}"  # grrrr
```
Comment (tkarna) [inline conda-recipe/build.sh:101]: rm comments?

**Review by tkarna — APPROVED** [review]
Looks good!

File: conda-recipe/build.sh (hunk 2 above)
Comment (fschlimb) [reply conda-recipe/build.sh:101]: If you don't mind I'll keep the comments for later when we enable oneAPI/devices.

**Pull request merged**